spec/spec_helper_acceptance.rb

```diff
@@ -40,15 +41,13 @@ def install_modules_on(host)
   options['is_puppetserver'] = true
   master['puppetservice'] = 'puppetserver'
   master['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
-  master['type'] = 'aio'
-  install_puppet_agent_on master, {}
-  install_package master, 'puppetserver'
```

Comment:
Is server installed by beaker-puppet now?

Reply:
puppet-server is not installed on master neither on host, as the server is not necessary in the context of these tests